src/components/shared/ButtonWithPanel/ConfirmDialog.js

```diff
@@ -60,13 +94,15 @@ export class ConfirmDialogImpl extends React.PureComponent<PropsWithClose> {
             type="button"
             className="photon-button photon-button-default"
             value={cancelButtonText || 'Cancel'}
+            disabled={status === 'working'}
```

Comment:
I'm not 100% sure we should disable the buttons. This gives an UX indication that we can't use them, but at the price of making them less visible. This is easy to revisit in the future though.